lib/metamodel.cto

```diff
@@ -224,6 +224,20 @@ concept ImportTypes extends Import {
   o String[] types
 }
 
+concept AliasType{
+  o String name
+  o String aliasName optional
+}
+
+// To discuss: Not required i think !! 
+// concept ImportAliasType extends Import {
+//   o AliasType name
+// }
+
+concept ImportAliasTypes extends Import {
+  o AliasType[] types
+}
+
```

Comment:
@ekarademir I think we don't need a separate concept for `importAliasType`, we can  work with `importAliasTypes` for the same. 

If we use the syntax [https://github.com/accordproject/concerto/issues/480#issuecomment-2134881794](here), we would only need single concept in both the cases i.e.importing multiple or single types